services/tables/src/test/resources/cluster-test-properties.yaml

```
@@ -2,6 +2,21 @@ cluster:
   name: "TestCluster"
   storage:
     root-path: "/tmp/unittest"
+  storages:
+    default-type: "hdfs"
+    types:
+      hdfs:
+        scheme: "hdfs"
+        rootpath: "/tmp/unittest"
+        endpoint: "hdfs://localhost:9000"
+        parameters:
+          key1: value1
+      objectstore:
+        scheme: "objectstorescheme"
+        rootpath: "tmpbucket"
+        endpoint: "http://localhost:9000"
+        parameters:
+          key2: value2
```

```suggestion
          token: xyz
```
Maybe we can add some keys we expect.